### ISSUE TITLE
Create new tidyqpcr vignette with MIQE compliant raw data

### ISSUE THREAD

**Issue by DimmestP** [ewallace/tidyqpcr#73]
The current delta-cq vignette shows how different forms of qPCR data can be incorporated and analysed by tidyqpcr but is not a good example of plate design according to the tidyqpcr ethos. A new vignette going from raw qpcr machine data to publishable results with MIQE compliant, tidy plate design would be a perfect stepping stone for new users.

**Comment by ewallace** [comment]
I agree this would be good. Let's design an experiment for this, just 2 conditions, 3 tech reps, 2 bio reps (or more), correct controls, that fits on a 96-well plate.

2 conditions * (3 tech reps + -RT 1 tech rep ) * 3 biol reps * 4 targets = 96 wells

**Comment by ewallace** [comment]
Reminder that we should design an experiment for this. We might even be able to ask @j-aux really nicely and she might do that experiment.

**Comment by ewallace** [comment]
There is a lot of old RNA extracted from Heat-shocked yeast (by me in 2018) in the -80 freezer. If that RNA is still ok, then we could use it. Or do a fresh "quick" experiment.
